src/scene/scene.js

```diff
@@ -551,6 +555,19 @@ class Scene extends EventHandler {
         return this._skyboxCubeMap;
     }
 
+    /**
+     * Type of sky. Defaults to {@link SKYTYPE_INFINITE}.
+     *
+     * @type {number}
+     */
+    set skyType(value) {
```

Comment:
we do not need to expose these new properties, those are already exposed as scene.sky.***
for example `skyType` is `scene.sky.type`

Reply:
and so the code that needs to access them just need to access them on `.sky`

Reply:
Ahh okay I wasnt sure if I should have forwarded the get call to this scene class but I will remove that